package.json

```diff
@@ -203,6 +203,7 @@
     "mkdirp": "^0.5.1",
     "pumpify": "^2.0.0",
     "tempfile": "^3.0.0",
+    "through2": "^3.0.1",
```

Comment:
is this used?

Reply:
Yes. I'm not sure how we were getting by without it except that vandelay already is using this.